index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header class="header">
+        <a class="logo">my logo</a>
+         <nav class="nav"><img src="./photos/Union.png" alt=""></nav>
+    </header>
+    <section class="intro">
+        <h1>I'm Theoneste Glad to see you!</h1>
+<p class="prg">
+    I’m a software developer! I can help you build a product ,
+     feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded, don’t 
+      hestiate to contact me.
+</p>
+
+<h4 class="connect">LET'S CONNECT</h4>
+<ul class="ul-list-icons">
```

Comment:
- [ ] Nice working on the social icons, kindly add (`a` tag) links to the icons
- [ ] Kindly re-size and align the social icons so they have the same height and width.
- [ ] Kindly use the same `font` provided in the Figma Template, so it aligns with what we have in the Figma Template

### Figma Template
![fig1](https://user-images.githubusercontent.com/48371156/170827211-fe3ef82c-7214-414e-b4a8-801f7bce6305.png)

### Your Design
![my_1](https://user-images.githubusercontent.com/48371156/170827146-756be036-23e6-4ae5-acc6-515bbd9444e2.png)